### PR TITLE
Update netsuite-connector.adoc | RESTLET Auth Info

### PR DESCRIPTION
**Pull request by jpestalardo**
I've taken this doc from the connector's repo. I see there are many differences between this one and mulesoft-docs version. For example: 
- toc is missing in mulesoft-docs
- icon also
Please let me know by this PR if we need to change it and I'll create a new pull request with those changes.
Thanks!

### CODE REVIEW

File: mule-user-guide/v/3.8/netsuite-connector.adoc [hunk 1]
```diff
@@ -25,7 +33,7 @@ For hardware and software requirements, please visit the link:/mule-user-guide/v
 
 === Compatibility
 
-For compatibility details for your version see the link:/release-notes/netsuite-connector-release-notes[release notes]
+For compatibility details for your version see the link:link:/release-notes/netsuite-connector-release-notes[release notes]
```
Comment (fermujica) [inline mule-user-guide/v/3.8/netsuite-connector.adoc]: Broken syntax: 
`link:link:/release-notes/netsuite-connector-release-notes[release notes]`

File: mule-user-guide/v/3.8/netsuite-connector.adoc [hunk 2]
```diff
@@ -49,15 +57,18 @@ To use the NetSuite connector in your Mule application, you must configure a
 
 This release of the connector lets you pick from a list of four Global Elements, each representing a different way of authenticating to NetSuite:
 
-[%header%autowidth.spread]
+[width="100%",cols="40%,60%",options="header"]
```
Comment (fermujica) [inline mule-user-guide/v/3.8/netsuite-connector.adoc]: We are using a different syntax for asciidoc tables now.

If you want to specify a length for the columns, the right syntax for this table is:
`[%header,cols="40a,60a"]`
Or you can just use the autowidth property and let asciidoc handle it.
`[%header%autowidth.spread]`

File: mule-user-guide/v/3.8/netsuite-connector.adoc [hunk 3]
```diff
@@ -1,18 +1,26 @@
 = NetSuite Connector
 :keywords: anypoint studio, esb, connector, endpoint, netsuite
 :imagesdir: ./_images
+:toc: macro
```
Comment (fermujica) [inline mule-user-guide/v/3.8/netsuite-connector.adoc]: We are getting rid of the TOC and the connector image from here.

**Comment by jpestalardo** [comment]
@martinpaoloni : could you review this in Netsuite documentation?

**Pull request merged**